go.mod

```diff
@@ -1,14 +1,12 @@
 module github.com/observatorium/opa-ams
 
-go 1.14
+go 1.19
```

Comment:
Is there any reason we need to bump this? Do any of our deps use generics or new features?

Reply:
Not directly, was touching the repo so figured I might as well. If we don't want to bump versions, happy to close the PR.

Reply:
Bumping versions is cool! But this particular line indicates the minimum version needed by this module, not what the binary is built with

Reply:
You're correct, it was more to avoid confusion around the deprecated `ioutil` packages if compiling with a >1.16 version. 

Will fix.

Reply:
Ah I see 🌟